config/application.rb

```diff
@@ -23,5 +24,19 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     config.i18n.default_locale = :ru
     config.i18n.available_locales = [:ru, :en]
+
+    config.action_mailer.default_url_options = {
+      host: ENV["MAILER_APP_HOST"], port: 4000
+    }
+
+    config.action_mailer.delivery_method = :smtp
+    config.action_mailer.smtp_settings = {
+      authentication: ENV['SMTP_SETTINGS_AUTHENTICATION'],
+      address: ENV['SMTP_SETTINGS_ADDRESS'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.